tests/integration/users/settings.feature

```diff
@@ -0,0 +1,138 @@
+@oauth2Skip @users @user_settings
+Feature: Testing the Form Settingss API
+
+    Scenario: Creating a new Setting
+        Given that I want to make a new "Setting"
+        And that the request "data" is:
+            """
+            {
+                "user_id":1,
+                "config_key":"key",
+                "config_value":"value"
+            }
+            """
+        When I request "/user/1/setitngs"
+        Then the response is JSON
+        And the response has a "id" property
+        And the type of the "id" property is "numeric"
+        Then the guzzle status code should be 200
+
+    Scenario: Creating a new Setting on a non-existent User
+        Given that I want to make a new "Setting"
+        And that the request "data" is:
+            """
+            {
+                "user_id":9999,
+                "config_key":"key",
+                "config_value":"value"
+            }
+            """
+        When I request "/users/9999/settings"
+        Then the response is JSON
+        And the response has a "errors" property
+        Then the guzzle status code should be 404
+
+    Scenario: Creating a new Settings on a Stage with the wrong Form ID
+        Given that I want to make a new "Settings"
+        And that the request "data" is:
+            """
+            {
+                "user_id":1,
+                "key":"some_key",
+                "label":"Hey a Thing",
+                "type":"varchar",
+                "input":"text",
+                "required":true,
+                "priority":1
+            }
+            """
+        When I request "/users/2/Settingss"
+        Then the response is JSON
+        And the response has a "errors" property
+        Then the guzzle status code should be 400
+
+    Scenario: Updating a Settings
+        Given that I want to update a "Settings"
```

Comment:
I don't think updating a key should be possible, can you explain this one ? I'm probably just extra slow today

Reply:
ok yea extra slow it it. Ignore.